### DISCUSSION TITLE
Clone a hyperFormulaInstance

### DISCUSSION THREAD

**Discussion by GabrielBrittoDev** [handsontable/hyperformula#852] [General]
How can i deep clone a hyperFormulaInstance?
I've tried so many methods and libraries, including serialization.

**Comment by MartinDawson** [comment]
Why do you want to deepClone an instance? That's almost certainly the wrong thing to do.

It's a class so it isn't serializalble.

Just get the serialized data using a hyperformula method such as `getAllSheetsSerialized()` and save that to your database or wherever.

Or just clone the options you pass in and start a new hyperformula instance and pass in your saved data.
